src/Components/Comment.js

```diff
@@ -1,26 +1,33 @@
 import axios from "axios";
 import { useEffect, useState } from "react";
 
+const apiUrlComment1 = "https://blogpostapi1.herokuapp.com/comment";
+const apiUrlComment2 = "https://blogpostapi1.herokuapp.com/comment/all_comments/";
```

Comment:
`apiUrlComment2` sounds more like `GET_ALL_COMMENTS_URL`.

And remember, you can just save `https://blogpostapi1.herokuapp.com` into a variable, instead of saving all your routes into a variable, like so...

```js
const apiUrl = "https://blogpostapi1.herokuapp.com";

axios.post(`${apiUrl}/comment`)

axios.get(`${apiUrl}/comment/all_comments`)
```